test/integration/connection_lost_test.exs

```diff
@@ -27,4 +27,37 @@ defmodule MongoosePush.API.ConnectionTest do
       Process.sleep(1000)
     end)
   end
+
+  test "Metrics can be correctly received from Prometheus" do
```

Comment:
We should make new test suite (file) for this test. It hardly qualifies as "connectio_lost_test".

Reply:
Also we need to disable this test (`@tag :skip`).